src/main/kotlin/uk/gov/justice/digital/hmpps/prisonregister/resource/PrisonAddressMaintenanceResource.kt

```diff
@@ -89,6 +91,64 @@ class PrisonAddressMaintenanceResource(
     )
     return updatedAddress
   }
+
+  @PreAuthorize("hasRole('ROLE_MAINTAIN_REF_DATA') and hasAuthority('SCOPE_write')")
```

Comment:
I am wondering if all these should have the REGISTER_MAINTAINER role.  Will check with rest of team

Reply:
Probably best as a separate ticket, depending on team view